editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -663,21 +655,10 @@ bool Sprite2DEditorPlugin::handles(Object *p_object) const {
 }
 
 void Sprite2DEditorPlugin::make_visible(bool p_visible) {
-	if (p_visible) {
-		sprite_editor->options->show();
-	} else {
-		sprite_editor->options->hide();
-		sprite_editor->edit(nullptr);
```

Comment:
The only thing that sets me off is the removal of this line, but I assume that's handled by something else.

Reply:
Yes, it's no longer necessary.